src/kernels/fixed_kernel.jl

```diff
@@ -1,6 +1,6 @@
 struct FixedKernel{K<:Kernel, NFREE} <: Kernel
     kernel::K
-    free::SVector{NFREE, Int64}
+    free::SVector{NFREE, Int}
```

Comment:
My reasoning was that having a generic type in a `struct` will slow things down. I haven't benchmarked it though…

Reply:
`Int` is not generic, if I understand [this](https://docs.julialang.org/en/v1/manual/integers-and-floating-point-numbers/#Integers-1) correctly.